code-challenges.test.js

```diff
@@ -24,51 +27,120 @@ var num2 = 0
+//     }
+ 
 
 
-// b) Create the function that makes the test pass.
 
 
+// // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+// // a) Create a test with expect statements for each of the variables provided.
 
+ //var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// / Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+//var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+//put inn describe and function name
+//establish your test
+//list expectations
+//  fdescribe("toUpperCase", () => {
+//      test("returns first letter of each word capitalized", () => {
+//         expect(toUpperCase(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         expect(toUpperCase(["temperature", "database", "chopsticks", "mango", "deduction"])).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+//      })
+//  })
 
-// a) Create a test with expect statements for each of the variables provided.
 
-var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 
-var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+// // // // b) Create the function that makes the test pass
+// // //make function to take in an array
+// // //make loop to capitalize each first letter of strings
+// // //return new array
+// // //log results
+// const toUpperCase = (array) => {
+//     return array.map((elem) => {
+//         return elem.charAt(0).toUpperCase() + elem.slice(1);
+//     })
+// }
 
+    
 
+ 
 
 
 
-// b) Create the function that makes the test pass.
 
 
 
 
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
-var vowelTester1 = "learn"
-// Expected output: 1
-var vowelTester2 = "academy"
-// Expected output: 0
-var vowelTester3 = "challenge"
-// Expected output: 2
+// // a) Create a test with expect statements for each of the variables provided.
+//establish definition
+//write in expected returns for each vowerltester
+ var vowelTester1 = "learn"
+// // // Expected output: 1
+  var vowelTester2 = "academy"
+// // // Expected output: 0
+  var vowelTester3 = "challenge"
+// // // Expected output: 2
+fdescribe("vowelFinder", () => {
```

Comment:
great testing suite. At a job if you are sending pull requests with tests make sure to drop the (f) so that they can see all the tests run together.